index.js

```diff
@@ -3,12 +3,23 @@
 var AWS = require('aws-sdk');
 var rc = require('rc');
 var config = rc('ec2-instance-lister');
-var searchString;
+var tagName = 'name';
+var tagValue;
 
 if (config.s) {
-	searchString = config.s;
+	tagValue = config.s;
+} else if (config.tag || config.t) {
+	config.tag = config.tag || config.t
```

Comment:
Ohh right sorry about that. Force if habit...
On Jan 25, 2016 5:09 PM, "aviramst" <notifications@github.com> wrote:

> In index.js
> <https://github.com/aviramst/ec2-instance-lister/pull/3#discussion_r50764322>
> :
>
> >
> >  if (config.s) {
> > -	searchString = config.s;
> > +	tagValue = config.s;
> > +} else if (config.tag || config.t) {
> > +	config.tag = config.tag || config.t
>
> Please add semicolons to be consistent with other lines on this file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aviramst/ec2-instance-lister/pull/3/files#r50764322>.
>